mergify_engine/tasks/engine/v2.py

```diff
@@ -141,6 +141,13 @@ def run_actions(installation_id, installation_token, subscription,
 def handle(installation_id, installation_token, subscription,
            pull_request_rules_raw, event_type, data, pull_raw):
 
+    # Some mandatory rules
```

Comment:
It makes the engine depends on a plugin (`delete_head_branch`).

Reply:
I have moved this into a configuration file.